gsw/geostrophy.py

```diff
@@ -184,8 +184,8 @@ def distance(lon, lat, p=0, axis=-1):
                           % (lon.shape, axis))
     if lon.ndim == 1:
         one_d = True
-        lon = lon[np.newaxis, :]
-        lat = lat[np.newaxis, :]
+        lon = np.expand_dims(lon, (0,))
```

Comment:
Since the arrays are 1-D, I suggest using the simpler `0` instead of `(0,)`, which will also then match the example in the numpy docs.  It might also be worth putting in a comment above this line: "xarray requires expand_dims() rather than [newaxis, :]"